packages/tools/devtools/devtools-core/src/messaging/container-devtools-messages/SendEditData.ts

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { HasContainerKey } from "../../CommonInterfaces";
+import { SharedObjectEdit } from "../../data-visualization";
+import { IDevtoolsMessage } from "../Messages";
+
+/**
+ * Encapsulates types and logic related to {@link SendEditData.Message}.
+ *
+ * @internal
+ */
+export namespace SendEditData {
```

Comment:
Nit: Since the name is intended to describe the message being sent, what do you think about "DataEdit" or something along those lines?